### PR TITLE
Add `vibrate-off` icon

### PR DESCRIPTION
**Pull request by lscheibel**
image

### CODE REVIEW

File: icons/vibrate-off.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m2 8 2 2-2 2 2 2-2 2" />
+  <path d="m22 8-2 2 2 2-2 2 2 2" />
+  <path d="M8 8v10c0 .55.45 1 1 1h6c.55 0 1-.45 1-1v-2" />
+  <path d="M16 10.34V6c0-.55-.45-1-1-1h-4.34" />
+  <path d="m2 2 20 20" />
```
Comment (karsa-mistmere) [inline icons/vibrate-off.svg:16]: @lscheibel Can you 
```suggestion
  <line x1="2" y1="2" x2="22" y2="22" />
```

**Comment by karsa-mistmere** [comment]
Nice one.

**Comment by lscheibel** [comment]
Changed the crossing line to use the line element

**Review by ericfennis — APPROVED** [review]
Nice icon! love it

**Pull request merged**